src/routes/admin/users/[userId]/+page.server.js

```diff
@@ -1,6 +1,7 @@
 import {getUserRoles, generateUuid} from "$lib/server/account.js";
 import {pool} from "../../../../hooks.server.js";
 import {redirect} from "@sveltejs/kit";
+import {getUserInfo, updateUser} from "../../../../lib/server/users.js";
```

Comment:
L'import raccourcir tu peux.
```suggestion
import {getUserInfo, updateUser} from '$lib/server/users.js';
```